### PR TITLE
Document new vmo UI flow 4.0

### PR DESCRIPTION
**Pull request by ritawatson**
## Describe the Change

This PR documents the new workflow integrated into Palette (using Public Repo), replaces mentions of "Spectro VM Dashboard pack" with "VMO pack", removes customer registry and license information because VMO is now available out-of-the-box, and adds OOB feature gates for Kubevirt. The PR also adds steps for evicting a VM using Palette.

This PR also adds a term to Vale and redirects for deleted pages.

**Affected pages:**

[Virtual Machine Orchestrator](https://deploy-preview-1461--docs-spectrocloud.netlify.app/vm-management) - Removed customer registry and license information and updated links as needed.
[Virtual Machine Orchestrator Pack](https://deploy-preview-1461--docs-spectrocloud.netlify.app/vm-management/vm-packs-profiles) - some clarification in the intro paragraph.
[Create a VMO Profile](https://deploy-preview-1461--docs-spectrocloud.netlify.app/vm-management/vm-packs-profiles/create-vmo-profile) - Documents the flow to create the profile using VMO pack.
[Add Roles and Role Bindings](https://deploy-preview-1461--docs-spectrocloud.netlify.app/vm-management/vm-packs-profiles/add-roles-and-role-bindings) - Previously, this doc was titled "Enable Spectro VM Dashboard". The doc is renamed because the Virtual Machines tab displays after profile creation.
[Standard VM Operations](https://deploy-preview-1461--docs-spectrocloud.netlify.app/vm-management/create-manage-vm/standard-vm-operations) - content from a deleted doc is now moved here.
[Evacuate a VM in Palette](https://deploy-preview-1461--docs-spectrocloud.netlify.app/vm-management/create-manage-vm/standard-vm-operations/migrate-vm-to-different-node#evacuatevmsinpalette)

💻 [Preview](https://deploy-preview-1461--docs-spectrocloud.netlify.app/vm-management)

🎫 [Jira Ticket](https://spectrocloud.atlassian.net/browse/PEM-2432)

### CODE REVIEW

File: content/docs/04.6-vm-management/05-vm-packs-profiles.md [hunk 1]
```diff
@@ -15,39 +15,37 @@ import InfoBox from 'shared/components/InfoBox';
...
+- **Spectro Proxy** pack: Enables the use of a reverse proxy with a Kubernetes cluster. This pack is automatically installed when you create the cluster with the default **Proxied** setting for **Access** during cluster profile creation. Check out the [Spectro Proxy](/integrations/frp) guide to learn more. 
 
 
-- **KubeVirt**: An add-on pack that allows you to create VMs within a Kubernetes cluster using open-source KubeVirt. KubeVirt provides feature gates you can enable in the Spectro VM Dashboard manifest. To learn which feature gates Palette enables by default and how you can enable additonal feature gates, check out the [Feature Gates](/vm-management#featuregates) section.
+- **KubeVirt**: Allows you to create VMs within a Kubernetes cluster using open-source KubeVirt. KubeVirt provides feature gates you can enable in the Virtual Machine Orchestrator pack YAML file. To learn which feature gates Palette enables by default and how you can enable additonal feature gates, check out the [Feature Gates](/vm-management#featuregates) section.
```
[the remark this answers is not in the document]
Comment (karl-cardenas-coding) [reply content/docs/04.6-vm-management/05-vm-packs-profiles.md:30]: @ritawatson Typo ☝🏻

File: content/docs/04.6-vm-management/05-vm-packs-profiles/15-create-vmo-profile.md [hunk 2]
```diff
@@ -46,40 +42,43 @@ Create a cluster profile with the **Spectro VM Dashboard** add-on pack and apply
...
-10. Configure the dashboard.
 
-<br />
+<WarningBox>
 
-## Configure the Dashboard
+We recommend using the pack defaults. Default settings provide best practices for your clusters. Changing the default settings can introduce misconfigurations. Carefully review the changes you make to a pack. 
 
-The default setting for **Access** is **Proxied**, which automatically adds the Spectro Proxy pack. Changing the default may require some additional configuration.
+</WarningBox>
```
[the remark this answers is not in the document]
Comment (ritawatson) [reply content/docs/04.6-vm-management/05-vm-packs-profiles/15-create-vmo-profile.md:67]: indented warning box

File: content/docs/04.6-vm-management/05-vm-packs-profiles/15-create-vmo-profile.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+- VMs with Persistent Volume Claim (PVC) must have a StorageClass that supports ``ReadWriteMany`` (``RWX``) access mode for seamless live migration to a different node - either when triggered manually or during a Kubernetes upgrades.
+
+
+- Outbound internet connectivity for port 443 is allowed so that you and your applications can connect with the Spectro Cloud reverse proxy.
+
+
+- Users or groups must be mapped to a Virtual Machine RBAC role. You can create a custom role through a manifest and use Palette's RoleBinding feature to associate users and groups with the role. Refer to the [Create Role Bindings](/clusters/cluster-management/cluster-rbac#createrolebindings) guide to learn more.
+
+
+- A namespace for VMs. Although you can deploy VMs from the default namespace, we recommend creating at least one namespace dedicated to VMs as a way to organize and manage them. To learn how to create a namespace, check out [Create a Namespace](/clusters/cluster-management/namespace-management#createanamespace).
```
Comment (karl-cardenas-coding) [inline content/docs/04.6-vm-management/05-vm-packs-profiles/15-create-vmo-profile.md:41]: Hm...I think these need to come up one level. These are more like global to the VMO feature. The prereqs for a cluster profile is to have the Palette permission key `create` for  the resource `clusterProfile`.

**Review by karl-cardenas-coding — APPROVED** [review]

**Review by tiwarisumit3 — APPROVED** [review]

File: content/docs/04.6-vm-management/05-vm-packs-profiles/15-create-vm-dashboard-profile.md [hunk 4]
```diff
@@ -1,107 +0,0 @@
----
```
Comment (cloudmaniac) [inline content/docs/04.6-vm-management/05-vm-packs-profiles/15-create-vm-dashboard-profile.md:1]: Why is this removed?

File: content/docs/04.6-vm-management/05-vm-packs-profiles/15-create-vmo-profile.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+
+    <br />
+    
+    - **Pack Type**: System App
+    - **Registry**: Public Repo
+    - **Pack Name**: Virtual Machine Orchestrator
+    - **Pack Version**: 1.0 or higher
+
+
+7. Review the **Access** configuration panel at right. The default setting is **Proxied**, which automatically adds the **Spectro Proxy** pack when you create the MAAS cluster. Check out the [Spectro Proxy](/integrations/frp) guide to learn more. Changing the default may require some additional configuration.
```
Comment (cloudmaniac) [inline content/docs/04.6-vm-management/05-vm-packs-profiles/15-create-vmo-profile.md:52]:
```suggestion
7. Review the **Access** configuration panel at right. The default setting is **Proxied**, which automatically adds the **Spectro Proxy** pack when you create the cluster. Check out the [Spectro Proxy](/integrations/frp) guide to learn more. Changing the default may require some additional configuration. 
```

File: content/docs/04.6-vm-management.md [hunk 6]
```diff
@@ -33,57 +31,27 @@ Palette VM Orchestrator is particularly suitable in the following scenarios:
 
 - Edge locations that have only a few VMs running, and a hypervisor is no longer desired.
 
-# Prerequisites
```
Comment (cloudmaniac) [inline content/docs/04.6-vm-management.md:36]: Why did all that section move to a different page? This page is the place where it should be.

File: content/docs/04.6-vm-management/05-vm-packs-profiles/15-create-vmo-profile.md [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+1.  Log in to [Palette](https://console.spectrocloud.com).
+
+
+2. Navigate to **Profiles** from the left **Main Menu**. 
+
+
+3. Locate the newly created profile in the list.
+
+
+4. From the left **Main Menu**, click **Clusters** and select your MAAS cluster.
```
Comment (cloudmaniac) [inline content/docs/04.6-vm-management/05-vm-packs-profiles/15-create-vmo-profile.md:103]:
```suggestion
4. From the left **Main Menu**, click **Clusters** and select your cluster.
```

File: content/docs/04.6-vm-management/05-vm-packs-profiles/15-create-vmo-profile.md [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+
+
+5. Based on your Single Sign-On (SSO) settings, the **Virtual Machines** tab may display on the **Cluster Overview** page, or the **Connect** button may display next to **Virtual Machines Dashboard** in cluster details.
```
Comment (cloudmaniac) [inline content/docs/04.6-vm-management/05-vm-packs-profiles/15-create-vmo-profile.md:106]: What is this "Connect" button? I don't have it.

File: content/docs/04.6-vm-management/05-vm-packs-profiles/20-add-roles-and-role-bindings.md [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+
+2. From the left **Main Menu**, click **Clusters** and select your cluster. 
+
+
+3. Navigate to the **Profiles** tab and click **+** next to **Addon Layers**.
+
+
+4. Select the VMO profile you created.
+
+
```
Comment (cloudmaniac) [inline content/docs/04.6-vm-management/05-vm-packs-profiles/20-add-roles-and-role-bindings.md:59]:
```suggestion
3. Navigate to the **Profiles** tab and click **+** next to **Addon Layers**.


4. Select the VMO profile you created.


```

File: content/docs/04.6-vm-management/10-create-manage-vm.md [hunk 10]
```diff
@@ -40,30 +64,22 @@ Additionally, Virtio is a virtualization standard for network and disk device dr
 
 <WarningBox>
 
-We recommend installing ``qemu-guest-agent`` to improve VM management. We also recommend installing Virtio drivers to ensure you can use the paravirtualized hardware properly.
+We recommend installing the QEMU guest agent to display additional details in Palette Virtual Machine Orchestrator. We also recommend installing VirtIO drivers to ensure you can use the paravirtualized hardware properly. To learn how to install the guest agent and drivers, refer to the [Red Hat](https://docs.openshift.com/container-platform/4.8/virt/virtual_machines/virt-installing-qemu-guest-agent.html) reference.
```
Comment (cloudmaniac) [inline content/docs/04.6-vm-management/10-create-manage-vm.md:67]: I'm not a big fan of linking competition, but I couldn't find a better page to link. :/
Also the steps on RedHat page only covers Windows and RedHat based OS; Debian, Ubuntu, etc., are not there.

File: content/docs/04.6-vm-management/10-create-manage-vm/05-standard-vm-operations/04-deploy-vm-from-template.md [hunk 11]
```diff
@@ -14,13 +14,13 @@ import InfoBox from 'shared/components/InfoBox';
...
 
-You can deploy a virtual machine (VM) using Palette's out-of-the-box templates or templates that your organization's administrator provides.
+You can deploy a Virtual Machine (VM) using Palette's out-of-the-box templates or from templates that your organization's administrator provides.
 
 # Prerequisites
 
 - Configured Spectro VM Dashboard profile applied to your cluster. 
 
-    Review [Create Spectro VM Dashboard Profile](/vm-management/vm-packs-profiles/create-vm-dashboard-profile) to configure the dashboard. To learn how to access the dashboard, refer to [Enable Spectro VM Dashboard](/vm-management/vm-packs-profiles/enable-vm-dashboard).
+    Review [Create a VMO Profile](/vm-management/vm-packs-profiles/create-vmo-profile) to configure the dashboard.
```
Comment (cloudmaniac) [inline content/docs/04.6-vm-management/10-create-manage-vm/05-standard-vm-operations/04-deploy-vm-from-template.md:23]:
```suggestion
Review [Create a VMO Profile](/vm-management/vm-packs-profiles/create-vmo-profile) to configure the dashboard.
```

File: content/docs/04.6-vm-management/10-create-manage-vm/05-standard-vm-operations/10-migrate-vm-to-different-node.md
Comment (cloudmaniac) [inline content/docs/04.6-vm-management/10-create-manage-vm/05-standard-vm-operations/10-migrate-vm-to-different-node.md:1]: Please change the `Spectro VM Dashboard` occurrence in this file (line 19).

**Review by cloudmaniac — COMMENTED** [review]
Nice job on the update. I'm done with my review, some comments to address but nothing major.

File: content/docs/04.6-vm-management.md [hunk 12]
```diff
@@ -33,57 +31,27 @@ Palette VM Orchestrator is particularly suitable in the following scenarios:
...
-The dashboard provides various methods to quickly deploy VMs from out-of-the-box templates or from your organization's templates. To learn more about using and creating templates, review [Deploy VM From a Template](/vm-management/create-manage-vm/standard-vm-operations/deploy-vm-from-template) and [Create a VM Template](/vm-management/create-manage-vm/create-vm-template). 
+Palette VM Orchestrator provides various methods to quickly deploy VMs from out-of-the-box templates or from your organization's templates. To learn more about using and creating templates, review [Deploy VM From a Template](/vm-management/create-manage-vm/standard-vm-operations/deploy-vm-from-template) and [Create a VM Template](/vm-management/create-manage-vm/create-vm-template). 
 
 
 # Feature Gates
 
-Palette VM Orchestrator utilizes open-source KubeVirt as a component of the **Spectro VM Dashboard** pack to manage VMs and enables the following KubeVirt feature gates by default:
+Palette VM Orchestrator utilizes open-source KubeVirt as a component of the **Virtual Machnine Orchestrator** pack to manage VMs and enables the following KubeVirt feature gates by default:
 
 <br />
```
[the remark this answers is not in the document]
Comment (ritawatson) [reply content/docs/04.6-vm-management.md:57]: Added new default feature gates and fixed VM Dashboard occurrence.

File: content/docs/04.6-vm-management/05-vm-packs-profiles.md [hunk 13]
```diff
@@ -15,39 +15,37 @@ import InfoBox from 'shared/components/InfoBox';
 
 # Overview
 
-Palette Virtual Machine Orchestrator (VMO) provides a single-pack experience that consolidates all the dependencies needed to deploy and manage VMs in your Kubernetes host cluster. 
+The **Virtual Machine Orchestrator** (VMO) pack provides a single-pack experience that consolidates all the dependencies needed to deploy and manage VMs in your Kubernetes host cluster. You use **Virtual Machine Orchestrator** pack to create a VMO cluster profile. the The pack's components are described below. All the components are enabled by default in the `charts:` section of the pack YAML configuration file.
```
[the remark this answers is not in the document]
Comment (ritawatson) [reply content/docs/04.6-vm-management/05-vm-packs-profiles.md:18]: fixed.

File: content/docs/04.6-vm-management/05-vm-packs-profiles.md [hunk 14]
```diff
@@ -15,39 +15,37 @@ import InfoBox from 'shared/components/InfoBox';
...
+The **Spectro Proxy** pack is a separate pack that is installed during profile creation when you use the default profile setting as described below.
 
 <br />
 
-- **Spectro VM Dashboard**: An add-on pack that enables access to a web console so you can manage and monitor your VMs. The console is accessible from the **Virtual Machines** tab that appears on the cluster overview page in Palette when VM Orchestrator is enabled. The dashboard provides a web interface to create and manage VMs in your Kubernetes cluster. To learn more about the web console, refer to [Spectro VM Dashboard](/vm-management/vm-packs-profiles/vm-dashboard).
+<!-- - **Spectro VM Dashboard**: An add-on pack that enables access to a web console so you can manage and monitor your VMs. The console is accessible from the **Virtual Machines** tab that appears on the cluster overview page in Palette when VM Orchestrator is enabled. The dashboard provides a web interface to create and manage VMs in your Kubernetes cluster. To learn more about the web console, refer to [Spectro VM Dashboard](/vm-management/vm-packs-profiles/vm-dashboard). -->
```
[the remark this answers is not in the document]
Comment (ritawatson) [reply content/docs/04.6-vm-management/05-vm-packs-profiles.md:24]: unhidden

File: content/docs/04.6-vm-management.md [hunk 15]
```diff
@@ -33,57 +31,27 @@ Palette VM Orchestrator is particularly suitable in the following scenarios:
 
 - Edge locations that have only a few VMs running, and a hypervisor is no longer desired.
```
[the remark this answers is not in the document]
Comment (ritawatson) [reply content/docs/04.6-vm-management.md:32]: Added.

File: content/docs/04.6-vm-management.md (hunk 6 above)
Comment (ritawatson) [reply content/docs/04.6-vm-management.md:36]: Discussed with @cloudmaniac  on a call. The reason content is moved is that with the new VMO pack, it no longer makes sense to have the Spectro VM Dashboard doc. The bullet list of tabs within the **Virtual Machines** tab now makes more sense in the Standard VM Operations doc.

File: content/docs/04.6-vm-management/05-vm-packs-profiles/15-create-vm-dashboard-profile.md (hunk 4 above)
Comment (ritawatson) [reply content/docs/04.6-vm-management/05-vm-packs-profiles/15-create-vm-dashboard-profile.md:1]: Discussed on a call with @cloudmaniac - VMO pack used to create VMO profile now replaces vm-dashboard profile.

File: content/docs/04.6-vm-management/05-vm-packs-profiles/15-create-vmo-profile.md [hunk 16]
```diff
@@ -0,0 +1,116 @@
...
+8. Click **Confirm & Create**. 
+
+
+9. In the following screen, click **Next**. 
+
+
+10. Review the profile and click **Finish Configuration**.
+
+
+11. Apply the profile to your MAAS cluster.
```
[the remark this answers is not in the document]
Comment (ritawatson) [reply content/docs/04.6-vm-management/05-vm-packs-profiles/15-create-vmo-profile.md:73]: removed

File: content/docs/04.6-vm-management/05-vm-packs-profiles/15-create-vmo-profile.md (hunk 5 above)
Comment (ritawatson) [reply content/docs/04.6-vm-management/05-vm-packs-profiles/15-create-vmo-profile.md:52]: removed

File: content/docs/04.6-vm-management/05-vm-packs-profiles/15-create-vmo-profile.md (hunk 7 above)
Comment (ritawatson) [reply content/docs/04.6-vm-management/05-vm-packs-profiles/15-create-vmo-profile.md:103]: removed

File: content/docs/04.6-vm-management/05-vm-packs-profiles/15-create-vmo-profile.md (hunk 8 above)
Comment (ritawatson) [reply content/docs/04.6-vm-management/05-vm-packs-profiles/15-create-vmo-profile.md:106]: @cloudmaniac 

@tiwarisumit3 showed me that the Connect button may appear instead of the Virtual Machines tab, depending on the user's SSO settings.

File: content/docs/04.6-vm-management/05-vm-packs-profiles/20-add-roles-and-role-bindings.md (hunk 9 above)
Comment (ritawatson) [reply content/docs/04.6-vm-management/05-vm-packs-profiles/20-add-roles-and-role-bindings.md:59]: Removed steps 3 and 4.

File: content/docs/04.6-vm-management/10-create-manage-vm.md (hunk 10 above)
Comment (ritawatson) [reply content/docs/04.6-vm-management/10-create-manage-vm.md:67]: @cloudmaniac Removing link to Redhat doc. How can we let users know how to install QEMU guest agent without documenting third-party software?

File: content/docs/04.6-vm-management/10-create-manage-vm/05-standard-vm-operations/04-deploy-vm-from-template.md [hunk 17]
```diff
@@ -102,7 +103,7 @@ VMs do not self-heal. If a VM is running on a node that fails, the VM is re-sche
 
 # Next Steps
 
-Try installing your applications. If the Guest Agent was not installed as part of the VM deployment, you can install the Guest Agent now. The Guest Agent displays additional details in the **Virtual Machines > Details** tab. 
+Try installing your applications. If you did not install the QEMU guest agent as part of the VM deployment, you can install it now. The guest agent displays additional details in the **Virtual Machines** > **Details** tab. To learn how to install the guest agent, review the [Red Hat](https://docs.openshift.com/container-platform/4.8/virt/virtual_machines/virt-installing-qemu-guest-agent.html) reference.
```
[the remark this answers is not in the document]
Comment (ritawatson) [reply content/docs/04.6-vm-management/10-create-manage-vm/05-standard-vm-operations/04-deploy-vm-from-template.md:106]: removed link. Per @cloudmaniac, we'll address this after 4.0.

File: content/docs/04.6-vm-management/10-create-manage-vm/10-standard-vm-operations.md [hunk 18]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "Standard VM Operations"
 metaTitle: "Standard VM Operations"
-metaDescription: "Learn about standard VM operations that can be performed using Spectro VM Dashboard."
+metaDescription: "Learn about standard VM operations that you can perform using Spectro VM Dashboard."
```
[the remark this answers is not in the document]
Comment (ritawatson) [reply content/docs/04.6-vm-management/10-create-manage-vm/10-standard-vm-operations.md:4]: fixed

File: content/docs/04.6-vm-management/10-create-manage-vm/10-standard-vm-operations.md [hunk 19]
```diff
@@ -15,7 +15,7 @@ import InfoBox from 'shared/components/InfoBox';
 
 # Overview
 
-The Spectro VM Dashboard supports standard power operations for virtual machines (VMs): 
+Virtual Machine Orchestrator supports standard power operations for Virtual Machines (VMs):
```
[the remark this answers is not in the document]
Comment (ritawatson) [reply content/docs/04.6-vm-management/10-create-manage-vm/10-standard-vm-operations.md:18]: added Palette.

File: content/docs/04.6-vm-management/10-create-manage-vm.md (hunk 10 above)
Comment (ritawatson) [reply content/docs/04.6-vm-management/10-create-manage-vm.md:67]: Per @cloudmaniac, removing the link to OpenShift doc. We'll address this as part of future VMO plans after 4.0.

File: content/docs/04.6-vm-management/10-create-manage-vm/05-standard-vm-operations/10-migrate-vm-to-different-node.md
Comment (ritawatson) [reply content/docs/04.6-vm-management/10-create-manage-vm/05-standard-vm-operations/10-migrate-vm-to-different-node.md:1]: fixed

File: content/docs/04.6-vm-management/15-vm-roles-permissions.md [hunk 20]
```diff
@@ -32,7 +32,7 @@ Palette provides the following four out-of-the-box Cluster roles for Palette VM
 
 <WarningBox>
 
-These roles are currently only relevant to access Palette VM Orchestrator APIs. To access the Virtual Machines console, users must have permissions to access the host clusters. These permissions can be granted through the [default Kubernetes roles](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#default-roles-and-role-bindings) Admin/Editor/Viewer.
+These roles are currently only relevant to access Palette Viritual Machine Orchestrator APIs. To access the Virtual Machines console, users must have permissions to access the host clusters. These permissions can be granted through the [default Kubernetes roles](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#default-roles-and-role-bindings) Admin/Editor/Viewer.
```
[the remark this answers is not in the document]
Comment (ritawatson) [reply content/docs/04.6-vm-management/15-vm-roles-permissions.md:35]: fixed

File: content/docs/04.6-vm-management/10-create-manage-vm/05-standard-vm-operations/15-take-snapshot-of-vm.md
Comment (cloudmaniac) [inline content/docs/04.6-vm-management/10-create-manage-vm/05-standard-vm-operations/15-take-snapshot-of-vm.md:1]: Please remove the link to the VMware documentation on quiescing; it's fine if we don't have the link.

File: content/docs/04.6-vm-management/15-vm-roles-permissions.md (hunk 20 above)
[the remark this answers is not in the document]
Comment (ritawatson) [reply content/docs/04.6-vm-management/15-vm-roles-permissions.md]: fixed typo

File: content/docs/04.6-vm-management/10-create-manage-vm/05-standard-vm-operations/15-take-snapshot-of-vm.md
Comment (ritawatson) [reply content/docs/04.6-vm-management/10-create-manage-vm/05-standard-vm-operations/15-take-snapshot-of-vm.md:1]: removed link

File: content/docs/04.6-vm-management/10-create-manage-vm/05-standard-vm-operations/15-take-snapshot-of-vm.md [hunk 21]
```diff
@@ -49,7 +49,7 @@ You can take a snapshot of an online VM that has hotplugged disks. Only persiste
 
 <WarningBox>
 
-In some situations, such as with the Fedora operating system, SELinux on the guest prevents the QEMU guest agent from [*quiescing*](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.hostclient.doc/GUID-64B866EF-7636-401C-A8FF-2B4584D9CA72.html?hWord=N4IghgNiBcII4FcCWBTAzgYxSAvkA) the target filesystem. As a workaround, you can do one of the following:
+In some situations, such as with the Fedora operating system, SELinux on the guest prevents the QEMU guest agent from quiescing the target filesystem. As a workaround, you can do one of the following:
```
[the remark this answers is not in the document]
Comment (ritawatson) [reply content/docs/04.6-vm-management/10-create-manage-vm/05-standard-vm-operations/15-take-snapshot-of-vm.md:52]: Resolved

[the remark this answers is not in the document]
Comment (ritawatson) [reply content/docs/04.6-vm-management/10-create-manage-vm/05-standard-vm-operations/15-take-snapshot-of-vm.md:52]: Resolved

**Review by cloudmaniac — APPROVED** [review]
Approved 👍
